README.md

```diff
@@ -135,14 +135,16 @@ Supported properties in the `config.*.json` files:
 | `api-version`             | `v2`      | TranSMART API version. Only `v2` is supported. |
 | `app-url`                 |           | URL where the Glowing Bear is accessible for the user.|
 | `autosave-subject-sets`   | `false`   | Persist subject selection as subject set automatically. |
-| `export-data-view`        | `default` | Shape of the export (`default`, `surveyTable`). |
+| `export-data-view`        | `default` | Shape of the export (`default`, `surveyTable`, `customFormat`). |
```

Comment:
The `customFormat` 'dataView' is now used to signal that the 'dataView' is not being used. Perhaps add a boolean flag somewhere else, or autodetect that if the `export-service-url` is set, the external service is used and 'dataView' can be ignored (but custom-export-job-name property is used).